<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>660</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>KubeConfigManager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets.qrc">
    <normaloff>:/logo/assets/logo.png</normaloff>:/logo/assets/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageWelcome">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEditContextInformation">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="markdown">
           <string>**Welcome**

</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButtonHomeSettings">
            <property name="text">
             <string>Settings</string>
            </property>
            <property name="icon">
             <iconset theme="preferences-system">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonHomeMergeFiles">
            <property name="text">
             <string>Merge configuration files</string>
            </property>
            <property name="icon">
             <iconset theme="edit-copy">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonHomeOpenDirectory">
            <property name="text">
             <string>Open folder</string>
            </property>
            <property name="icon">
             <iconset theme="folder-open">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageConfigEdit">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelContexts">
                <property name="text">
                 <string>Contexts</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QListView" name="listViewContexts">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelWorkingDirectory">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Working Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditWorkingDirectory">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonWorkingDirectory">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="system-search">
                  <normaloff>.</normaloff>.</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QGroupBox" name="groupBoxSelectedContext">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Selected context</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="labelContextName">
                   <property name="text">
                    <string>Context name:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditContextName"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="labelCluster">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Cluster:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBoxCluster">
                   <property name="editable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="labelUser">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>User</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBoxUser">
                   <property name="editable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageConfigMerge"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetConfigurationFiles">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Configuration Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListView" name="listViewFiles">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::ActionMask</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButtonMergeFiles">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Merge files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDeleteFile">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="pushButtonCloneFile">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRenameFile">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Rename</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="system-shutdown">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionSwitchContext">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Switch context</string>
   </property>
   <property name="toolTip">
    <string>Switch to another context</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionEditUsers">
   <property name="icon">
    <iconset theme="contact-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Edit users</string>
   </property>
  </action>
  <action name="actionEditClusters">
   <property name="icon">
    <iconset theme="applications-engineering">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Edit clusters</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionNewKubeConfigFile">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New KubeConfig file</string>
   </property>
   <property name="toolTip">
    <string>Start a config file from scratch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionToggleFilesPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="view-sort-ascending">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Toggle files Panel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionStackedWidgetHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionStackedWidgetMerger">
   <property name="text">
    <string>Merger</string>
   </property>
  </action>
  <action name="actionStackedWidgetConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections/>
</ui>
